parsons/aws/s3.py

```diff
@@ -177,10 +173,12 @@ def list_keys(
                 resp = self.client.list_objects_v2(**args)
 
             except ClientError as e:
-                logger.error("FAILED TO RETURN OBJECTS!")
-                logger.error(
-                    "Check your permissions in this bucket, and consider providing a prefix"
-                )
+                error_message = """Unable to list bucket objects!
+                This may be due to a lack of permission on the requested
+                bucket. Double-check that you have sufficient READ permissions
+                on the bucket you've requested"""
```

Comment:
```suggestion
                error_message = """Unable to list bucket objects!
                This may be due to a lack of permission on the requested
                bucket. Double-check that you have sufficient READ permissions
                on the bucket you've requested. If you only have permissions for
                keys within a specific prefix, make sure you include a trailing '/' in
                in prefix."""
```

Reply:
I think I have a typo but you get the idea